src/main/java/ru/itmo/roguelike/field/Tile.java

```diff
@@ -29,7 +29,7 @@ public Tile() {
     /**
      * Creates empty tile and initialized it.
      *
-     * @param value real number in range [0, 1]. Needed to compute {@see TileType}.
+     * @param value real number in range [0, 1]. Needed to compute (see {@link TileType}).
```

Comment:
I think, word "see" is redundant here. Because it is pretty convenient to read the sentence as "Needed to compute TileType"

Reply:
fixed